### PR TITLE
short intro section on dynamic plugins

### PR DESCRIPTION
**Pull request by rsriniva**

### CODE REVIEW

File: modules/chapter2/pages/dynamic.adoc [hunk 1]
```diff
@@ -1,3 +1,53 @@
 = Dynamic Plugins
 
-NOTE: This section is under development...
\ No newline at end of file
+This section briefly explains the rationale for dynamic plugins and how to enable or disable plugins in RHDH.
+
+== Dynamic Plugins in RHDH
+
+Ass you already learnt in the first introductory course in the learning path, RHDH, as a platform consists of a small modular core, and an extensive set of plugins that provide most of the functionality. In the upstream *Backstage.io* platform, enabling a plugin means modifying the source code of the platform to load a bunch of TypeScript/JavaScript, and other static files (HTML, CSS, images etc.). You then have to stop and restart the runtime for the plugin to be enabled.
```
Comment (jayfray12) [inline modules/chapter2/pages/dynamic.adoc:7]: Type in first word :)

File: modules/chapter2/pages/dynamic.adoc [hunk 2]
```diff
@@ -1,3 +1,53 @@
...
+      - disabled: false <1>
+        package: >- <2>
+          ./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-dynamic
+```
+
+<1> Enable or Disable the plugin. Set to `true` to disable the plugin
+<2> The path to the dynamic plugin. Consult the `Administration Guide` (dynamic plugins matrix) for the full path
+
+WARNING: Simply enabling a dynamic plugin in the helm chart may not provide the required functionality. You may need to edit your *app-config.yaml* files to add plugin specific configuration (for example, set different environment variables) to make the plugin work correctly. Consult the corresponding plugin documentation for the required configuration.
+
```
Comment (jayfray12) [inline modules/chapter2/pages/dynamic.adoc:49]: Does it make sense to add a section about hosting a dynamic plugin in a npm repository or is that outside the scope of this specific module?

Comment (rsriniva) [reply modules/chapter2/pages/dynamic.adoc:49]: yeah good one. found this on the RHDH slack channel - https://www.youtube.com/watch?v=yVzjTU3JamE

File: modules/chapter2/pages/dynamic.adoc (hunk 1 above)
Comment (rsriniva) [reply modules/chapter2/pages/dynamic.adoc]: fixed

File: modules/chapter2/pages/dynamic.adoc (hunk 2 above)
Comment (rsriniva) [reply modules/chapter2/pages/dynamic.adoc:53]: fixed. added links for npm registry hosted plugins

**Pull request merged**